src/webgpu/api/operation/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,24 @@
+export const description = `API Operation Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that a render pass with color attachment storeOp has correct output for combinations of:
+      - storeOp set to {'clear', 'store', 'undefined}
+      - All color renderable formats
+      - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS'
+      - mip level set to {'0', mip > '0'}
+      - array layer set to {'1', layer > '1'}
+      - depth slice set to {'0', slice > '0'}
+
+  - Test that a render pass with a depthStencilAttachment has correct output for combinations of:
+      - depthStoreOp set to {'clear', 'store', 'undefined'}
+      - stencilStoreOp set to {'clear', 'store', 'undefined'}
+      - All depth/stencil formats
+      - mip level set to {'0', mip > '0'}
+      - array layer set to {'0', layer > '0'}
+      - depth slice set to {'0', slice > '0'}`;
+
```

Comment:
also test that using color attachments and depthStencil attachments together work correctly,

and test using color attachments with different store ops as they should clear/store independently

Reply:
Done. I think this has now might be best done in a single test with a giant param list.